src/Maui/Features/Combat/Manage.razor.cs

```diff
@@ -184,7 +198,7 @@ public void EndCombatEncounter()
 
         public bool IsCreaturesTurn(InitiatedCreature creature) => _currentTurnCreature == creature;
 
-        public bool IsCombatActive() => _currentTurnCreature is not null;
+        public bool IsCombatActive() => _initiatedCreatures.Any();
```

Comment:
I dont think combat is active if the list has things in it. That’ll probably make sense if Add() wasn’t populating it.

Maybe Add() shouldn’t populate that list, instead keeping the creatures until combat is started?

Reply:
See PR #70